R/reshape.R

```diff
@@ -31,15 +32,19 @@ du.reshape <- function(upload = TRUE, project, data_version, input_format, dict_
   nonrep_data <- du.reshape.generate.non.repeated(
     data, dict_kind
   )
+  #if (!is.null(nonrep_data)) {write_csv(nonrep_data, paste0(getwd(), "/", file_name_nonrep, ".csv"), na = "")} 
   if (!is.null(nonrep_data)) write_csv(nonrep_data, paste0(getwd(), "/", file_name_nonrep, ".csv"), na = "")
   yearlyrep_data <- du.reshape.generate.yearly.repeated(
-    data, dict_kind
+   data, dict_kind
   )
   if (!is.null(yearlyrep_data)) write_csv(yearlyrep_data, paste0(getwd(), "/", file_name_yearly, ".csv"), na = "")
-  monthlyrep_data <- du.reshape.generate.monthly.repeated(
-    data, dict_kind
-  )
-  if (!is.null(monthlyrep_data)) write_csv(monthlyrep_data, paste0(getwd(), "/", file_name_monthly, ".csv"), na = "")
+  #monthlyrep_data <- du.reshape.generate.monthly.repeated(
+  # data, dict_kind
+  #)
```

Comment:
Do we remove the reshape for the monthly rep altogether? is this what we want?

Reply:
uncommented the code and replaced is.null by exists